lib/breakpointhook.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function() {
```

Comment:
Done adding a comment. I don't think this belongs under debugger/ because all of the code in there runs in the separate debugger process, whereas this is part of the same process as browser + friends.